src/openklant/components/klantinteracties/api/serializers/partijen.py

```diff
@@ -454,11 +457,8 @@ class PartijSerializer(NestedGegevensGroepMixin, PolymorphicSerializer):
             "Digitaal adres waarop een partij bij voorkeur door de gemeente benaderd wordt."
         ),
     )
-    vertegenwoordigde = PartijForeignKeySerializer(
-        required=True,
-        allow_null=True,
-        many=True,
-        help_text=_("Partij die een andere partij vertegenwoordigde."),
+    vertegenwoordigden = serializers.SerializerMethodField(
```

Comment:
Dit komt er uit als string maar moet een array zijn van Partijen.

Denk dat dit gewoon PartijFKSerializer moet zijn zoals erboven is weggehaald.